### ISSUE TITLE
WhatsApp's 24-hour red message does not appear for agents.

### ISSUE THREAD

**Issue by sebavieira** [sendingtk/chatwoot#79]
Hello everyone, I hope you are all doing well.

Recently, I noticed that users with agent-level access are not able to see the 24-hour red stripe on WhatsApp.

I use it here at the company, and some users have reported this to me.

My version: v3.10.3

**Comment by matisidler** [comment]
Yep, same here, but in my case I'm unable to see it even with admin access 👀
